commons/src/test/java/com/simpletools/calculator/commons/CalculatorImplTest.kt

```diff
@@ -120,9 +119,7 @@ class CalculatorImplTest {
         calc.handleOperation("plus")
         calc.addDigit(2)
         calc.handleClear()
-        assertEquals(activity.btn_clear.text,"AC")
```

Comment:
Good, don't want to test too many things in one test.

Reply:
I didn't want to test UI stuff in a unit test also, the behavior is already tested. it is not the value of a string that will confirm the behavior.